tests/scripts/generate_psa_tests.py

```diff
@@ -299,13 +314,12 @@ def key_for_lifetime(
                        r'', short)
         short = re.sub(r'PSA_KEY_[A-Z]+_', r'', short)
         description = 'lifetime: ' + short
-        key = StorageKey(version=self.version,
-                         id=1, lifetime=lifetime,
-                         type='PSA_KEY_TYPE_RAW_DATA', bits=8,
-                         usage='PSA_KEY_USAGE_EXPORT', alg=0, alg2=0,
-                         material=b'L',
-                         description=description)
-        return key
+        return StorageKey(version=self.version,
```

Comment:
Minor: there is an advantage to
```
variable = complicated_expression()
return variable
```
rather than
```
return complicated_expression()
```
which is that it's easier to set a breakpoint and examine the result of `complicated_expression()` before the function returns.